sites/friday/src/get-started/laravel/deploy/deploy.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Deploy Laravel"
+sidebarTitle: "Deploy"
+weight: -800
+description: |
+  Now that your Laravel app is ready, push it to {{< vendor/name >}} and import your data.
+---
+{{% guides/deployment %}} 
```

Comment:
because we never instructed the user to set the remote branch, if you attempt to do `git push` per the instructions, you get a git error:
```
$  git push                                                                                                         
fatal: The current branch main has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream upsun main
```
Since we never instructed the user to commit, you get the error that there is no configuration file.

Reply:
We did `upsun project:set-remote` earlier. Wasn't that supposed to set the remote as upstream?